app/views/admin/reports/show.html.haml

```diff
@@ -16,8 +16,8 @@
     %legend{ align: 'center'}= t(:report_render_options)
     = render partial: "rendering_options"
 
-    .actions.filter-actions
-      = button t(:go), "report__submit-btn"
+    #report-go.actions.filter-actions{ data: { controller: "scoped-channel", "scoped-channel-id-value": request.uuid } }
+      = button t(:go), "report__submit-btn", "submit", "data-disable-with": true 
```

Comment:
Oh, I didn't pick up on this before. The `data-disable-with` attribute is used to replace the button text. 
Usually, rails magic JS will auto-populate this, but I guess you found that you had to manually include it.

I think we can simply include the go text here:

```suggestion
      = button t(:go), "report__submit-btn", "submit", "data-disable-with": t(:go) 
```

Reply:
look like I miss read the documentation it should have been : 
```suggestion
      = button t(:go), "report__submit-btn", "submit", "data-disable": true
```
I'll fix that quickly